src/Features/Core/Portable/OrderModifiers/AbstractOrderModifiersCodeFixProvider.cs

```diff
@@ -33,8 +33,10 @@ internal abstract class AbstractOrderModifiersCodeFixProvider : SyntaxEditorBase
             _helpers = helpers;
         }
 
-        public override ImmutableArray<string> FixableDiagnosticIds { get; } =
-            ImmutableArray.Create(IDEDiagnosticIds.OrderModifiersDiagnosticId);
+        protected abstract ImmutableArray<string> FixableCompilerErrorIds { get; }
+
+        public sealed override ImmutableArray<string> FixableDiagnosticIds
+            => FixableCompilerErrorIds.Add(IDEDiagnosticIds.OrderModifiersDiagnosticId);
```

Comment:
i'm a little concerned about this.  Previously, we know the IDE was generating the diagnostic, so we could trust the location of the diagnostic we were then fixing.  We don't control that with the compiler, and so using hte diagnostic without validation can be risky.  Specifically, for code like: 

> var memberDeclaration = diagnostic.Location.FindNode(cancellationToken);

We should actually validate we're on a proper node, and bail out if not.

Reply:
Fixed in 2d3144d46883584277844b7cbf8480fb573739fd